backend/src/errors/AuthenticationError.ts

```diff
@@ -0,0 +1,9 @@
+export class AuthenticationError extends Error {
```

Comment:
Is there a reason the AuthenticationError is placed in a separate file, whereas the others are all in the same file?

Reply:
No, not really, I merged the files, see [2a8571e](https://github.com/formcapture/form-backend/pull/164/commits/2a8571e50e185804df64c65f513deebccf31e484)